src/components/tree/phyloTree/phyloTree.ts

```diff
@@ -8,9 +8,89 @@ import * as grid from "./grid";
 import * as confidence from "./confidence";
 import * as labels from "./labels";
 import * as regression from "./regression";
+import { Callbacks, Layout, Params, PhyloNode, ReduxNode, ScatterVariables, Visibility } from "./types";
+import { Selection } from "d3";
+
+export interface PhyloTree {
```

Comment:
why not put this in `./types`?

Reply:
Done: https://github.com/nextstrain/auspice/blob/3cf6768919650f4294a8c7a213cfa11dda5fc462/src/components/tree/phyloTree/types.ts#L211